src/helix/client/client_ext.rs

```diff
@@ -845,6 +845,28 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
         }
     }
 
+    /// Get the emotes of the user associated with this token.
+    pub fn get_user_emotes<T>(
+        &'client self,
+        token: &'client T,
+    ) -> impl futures::Stream<Item = Result<helix::chat::UserEmote, ClientError<C>>>
```

Comment:
need to add a "in channel" function also, or maybe make this one take an optional broadcaster_id